src/styles/RenderRocket.scss

```diff
@@ -23,20 +36,51 @@
       margin: 15px 0;
     }
 
-    button {
+    .reserveBtn {
       background-color: $blue-color;
       padding: 10px 20px;
       border-radius: 5px;
       border: 1px solid #fff;
       color: #efefef;
     }
```

Comment:
[OPTIONAL] I think it would be a really good idea to increment the font size of the buttons as well as their weight. It would make it look better for desktop screens.